py/client/demo_simple.py

```diff
@@ -0,0 +1,56 @@
+from __future__ import annotations
+
+import pydeephaven as pyd
+import threading
+import time
+from pydeephaven.table import Table
+from pydeephaven.ticking.table_listener import TableListener, TableUpdate, TableListenerHandle
```

Comment:
here we have a sub-namespace `ticking`, I don't necessarily think it is bad, but in the spirit of keeping consistency between the server and client API, is it possible to make it a private sub-package and import it directly into pydeephaven so that it can be used as a top level module?

Reply:
I think I need your help on this. My objective was to set things up so that the ticking piece would be optional. That is, if the customer didn't want to run the Cython/C++ build step, and their Python code didn't access any "ticking" functions, things would just work fine.

I don't really know how to set things up, and how to write build/install instructions, so that the ticking piece can be truly optional.

Reply:
OK, thanks to Jianfeng's work these are now two different packages: pydeephaven and pydeephaven-ticking.